### PR TITLE
Upgrade to elasticsearch 6.6.0

### PR DESCRIPTION
**Pull request by bcbrockway**
* Fixes for [breaking changes](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/breaking-changes-6.5.html#breaking_65_logging_changes)
* Made jdk folder changes version-agnostic
* Some updates to examples

### CODE REVIEW

File: examples/minikube/README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ You might want to review those settings, especially if you are lucky enough to h
 
 * kubectl apply -f namespace.yml
 * kubectl apply -f config.yml
+* kubectl apply -f rbac.yml
```
Comment (primeroz) [inline examples/minikube/README.md]: Does it fail if you don't have RBAC enabled in your minikube ?

File: run.sh [hunk 2]
```diff
@@ -134,11 +134,11 @@ fi
 
 ## DNS Timers
 if [ ! -z "${NETWORK_ADDRESS_CACHE_TTL}" ]; then
-    sed -i -e "s/#networkaddress.cache.ttl=.*/networkaddress.cache.ttl=${NETWORK_ADDRESS_CACHE_TTL}/" /opt/jdk-10.0.2/conf/security/java.security
+    sed -i -e "s/#networkaddress.cache.ttl=.*/networkaddress.cache.ttl=${NETWORK_ADDRESS_CACHE_TTL}/" /opt/jdk-*/conf/security/java.security
```
Comment (primeroz) [inline run.sh:137]: Let's make srure this work as expected

File: examples/minikube/README.md (hunk 1 above)
Comment (bcbrockway) [reply examples/minikube/README.md]: Yeah, complains that it can't find the service account.

File: run.sh (hunk 2 above)
Comment (bcbrockway) [reply run.sh:137]: All looks good:

```
[elasticsearch@18d8ff7c8f7f security]$ grep ttl java.security 
networkaddress.cache.ttl=3
networkaddress.cache.negative.ttl=10
```

**Review by primeroz — APPROVED** [review]

**Pull request merged**